cs464-group-project/src/pages/Team.js

```diff
@@ -1,3 +1,27 @@
+import React, { useState, useEffect } from 'react';
+import { fetchData } from '../components/api/Api';
```

Comment:
`components/api/Api` needs to be changed to `component/Api/Api` or folder needs to be renamed.

Reply:
I renamed the file to avoid confustion.  Tested and no issues
![image](https://github.com/CS464-Group-Project/CS464_GroupProject/assets/97830813/0b465645-9c54-46a6-a462-272e46d9e73c)